src/Symfony/Component/DependencyInjection/Argument/AbstractableArgument.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Argument;
+
+/**
+ * Represents an abstract parameter, which have to be set by a compiler pass or a DI extension.
```

Comment:
an abstract service argument ("parameter" is confusing with DI params)

Reply:
done